src/Microsoft.IdentityModel.Tokens/Validators.cs

```diff
@@ -223,7 +224,7 @@ public static string ValidateIssuer(string issuer, SecurityToken securityToken,
                     { InvalidIssuer = issuer });
 
             // Throw if all possible places to validate against are null or empty
-            if (string.IsNullOrWhiteSpace(validationParameters.ValidIssuer) && (validationParameters.ValidIssuers == null))
+            if (string.IsNullOrWhiteSpace(validationParameters.ValidIssuer) && (validationParameters.ValidIssuers == null) && (validationParameters.Configuration?.Issuer == null))
```

Comment:
Should this be changed to the following?
            if (string.IsNullOrWhiteSpace(validationParameters.ValidIssuer) && (validationParameters.ValidIssuers == null || !validationParameters.ValidIssuers.Any()) && (string.IsNullOrWhiteSpace(validationParameters.Configuration?.Issuer)))

Reply:
Good catch! I added the empty check for ValidIssuers here as well.

Reply:
We should check the Issuer with string.IsNullOrWhiteSpace(validationParameters.Configuration?.**Issuer**) so it is consistent with the string.IsNullOrWhiteSpace(validationParameters.**ValidIssuer**) check.

Reply:
Got it, fixed! https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1695/commits/f5b3504e5f237e293dce12f52e01e133ace89fe3